src/schemas/ingestion/inputFiles.schema.ts

```diff
@@ -1,11 +1,35 @@
 import { z } from 'zod';
 import { INGESTION_VALIDATIONS } from '../../constants/ingestion/constants';
 
+export const gpkgFileNameSchema = z
+  .string()
+  .regex(new RegExp(INGESTION_VALIDATIONS.fileNames.pattern), 'File name must end with .gpkg')
+  .describe('gpkgFileNameSchema');
+
+export const metadataShapefilePathSchema = z
+  .string()
+  .regex(
+    new RegExp(INGESTION_VALIDATIONS.metadataShapefileFilePath.pattern),
+    'Metadata shape file path must be a valid shapefile path ending with "/ShapeMetadata.shp"'
```

Comment:
move this description to constants.ts into INGESTION_VALIDATIONS.metadataShapefileFilePath.description